build.gradle

```diff
@@ -76,11 +76,7 @@ subprojects {
     }
 
     dependencies {
-        ballerinaStdLibs "io.ballerina.stdlib:io-ballerina:${stdlibIoVersion}"
```

Comment:
Lets remove the entries from gradle.properties too?

Reply:
updated in [bc26c68](https://github.com/ballerina-platform/persist-tools/pull/37/commits/bc26c68c6558181a0eca576fe4137839b0b6b741)